File: 30_days_of_code/day26/day26.py
#!/bin/python3

[day, month, year] = map(int, input().split(' '))
[dayExpected, monthExpected, yearExpected] = map(int, input().split(' '))

fine = 0
if(year > yearExpected):
    fine = 10000

elif(year == yearExpected):
    if(month > monthExpected):
        fine = 500 * (month - monthExpected)

    elif((month == monthExpected) and (day > dayExpected)):
        fine = 15 * (day - dayExpected)

    elif((month == monthExpected) and (day == dayExpected)):
        fine = 0

print(fine)

File: 30_days_of_code/day26/day26.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
  int day, month, year;
  scanf("%d%d%d", &day, &month, &year);

  int dayExpected, monthExpected, yearExpected;
  scanf("%d%d%d", &dayExpected, &monthExpected, &yearExpected);

  int fine = 0;
  if (year > yearExpected) {
    fine = 10000;
  }
  else if (year == yearExpected) {
    if (month > monthExpected) {
      fine = 500 * (month - monthExpected);
    }
    else if ((month == monthExpected) && (day > dayExpected)) {
      fine = 15 * (day - dayExpected);
    }
    else if ((month == monthExpected) && (day == dayExpected)) {
      fine = 0;
    }
  }

  printf("%d\n", fine);

  return 0;
}
